README.rst

```diff
@@ -1,76 +1,80 @@
-|Build Status| |Codecov| |license|
+|Build Status| |license|
 
-frontend-template-application
-=================================
+frontend-app-skills
+==============================
 
-Please tag **@edx/fedx-team** on any PRs or issues.  Thanks.
+Please tag **@edx/edx-aperture** on any PRs or issues.
 
 Introduction
 ------------
 
-This repository is a template for Open edX micro-frontend applications. It is flagged as a Template Repository, meaning it can be used as a basis for new GitHub repositories by clicking the green "Use this template" button above.  The rest of this document describes how to work with your new micro-frontend after you've created a new repository from the template.
+This is a micro-frontend application responsible for displaying and maintaining tools for skills development. This is the home for the B2C Skills Builder. 
 
-After Copying The Template
---------------------------
+.. note::
+   This MFE is currently under development. Aperture is in the process of extracting the Skills Builder from the Profile MFE to this location. See the `edX Skills Builder - Development Roadmap`_.
```

Comment:
I hope we remember to come back and update this note as the dynamics change!

Reply:
I've removed this seeing as we have the link to the roadmap already included below. The thinking: anyone who is curious about the development will know to scan the rest of the README. Happy to put a note though if we think it's still necessary